packages/eslint-plugin-svelte/src/rules/no-navigation-without-resolve.ts

```diff
@@ -263,37 +261,37 @@ function expressionIsEmpty(url: TSESTree.CallExpressionArgument): boolean {
 	);
 }
 
-function expressionIsAbsolute(
+function expressionIsIgnored(
 	ctx: FindVariableContext,
 	url: AST.SvelteLiteral | TSESTree.Expression
 ): boolean {
 	switch (url.type) {
 		case 'BinaryExpression':
-			return binaryExpressionIsAbsolute(ctx, url);
+			return binaryExpressionIsIgnored(ctx, url);
 		case 'Identifier':
-			return identifierIsAbsolute(ctx, url);
+			return identifierIsIgnored(ctx, url);
 		case 'Literal':
-			return typeof url.value === 'string' && urlValueIsAbsolute(url.value);
+			return typeof url.value === 'string' && urlValueIsIgnored(url.value);
 		case 'SvelteLiteral':
-			return urlValueIsAbsolute(url.value);
+			return urlValueIsIgnored(url.value);
 		case 'TemplateLiteral':
-			return templateLiteralIsAbsolute(ctx, url);
+			return templateLiteralIsIgnored(ctx, url);
 		default:
 			return false;
 	}
 }
 
-function binaryExpressionIsAbsolute(
+function binaryExpressionIsIgnored(
 	ctx: FindVariableContext,
 	url: TSESTree.BinaryExpression
 ): boolean {
 	return (
-		(url.left.type !== 'PrivateIdentifier' && expressionIsAbsolute(ctx, url.left)) ||
-		expressionIsAbsolute(ctx, url.right)
+		(url.left.type !== 'PrivateIdentifier' && expressionIsIgnored(ctx, url.left)) ||
+		expressionIsIgnored(ctx, url.right)
```

Comment:
Beware that this would change the behaviour for the expression `"a" + "#b"` which currently isn't ignored but would be in this PR